examples/kebechet/alerting.md

```diff
@@ -0,0 +1,24 @@
+# Alerting
+
```

Comment:
can we add at the beginning:

"Alerting on metrics is essential to monitoring. They allow you to spot problems anywhere in your infrastructure, application, etc so that you can rapidly identify their causes and minimize service degradation and disruption. An alert should communicate something specific about your systems such as “90% of all web requests are taking more than 0.5s to process and respond." Receiving these alerts allows you to respond quickly to issues and provide better service, and it also saves time by freeing you from continual manual inspection of metrics.

You can get notified of these alerts......"